src/Classes/ServiceAPI/MyRadio_Show.php

```diff
@@ -247,7 +247,7 @@ public static function create($params = []) {
         if (!empty($_SESSION['memberid'])) {
             $creator = $_SESSION['memberid'];
         } else {
-            $creator = $params['credits'][0]['memberid'];
+            $creator = $params['credits']['memberid'][0];
```

Comment:
Yeah, that is right - it's an artifact of the data model being what MyRadioForm sends back.